.github/workflows/rust.yml

```diff
@@ -19,11 +19,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: setup submodules
+        run: git submodule update --init --recursive
+
       - name: Cancel previous runs
         uses: styfle/cancel-workflow-action@0.5.0
         with:
           access_token: ${{ github.token }}
 
+      - name: Setup apt packages
+        run: sudo apt-get update && sudo apt-get install -y cmake llvm-dev clang libclang-dev
```

Comment:
`apt-get install -y --no-install-recommends`

Reply:
didn't know about this one! thanks